en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+# Cocos Creator v2.1.0 Release Notes
+
+__Cocos Creator__ __v2.1__, adds support for 3D features such as: **3D model rendering**, **3D Camera**, **3D skeletal animation**, and **3D point selection**. The __Cocos Creator__ editor now natively supports parsing 3D model files in **FBX format**. It's that simple, no additional workflow is required!
+
+In the previous version, v2.0, __Cocos Creator__ had used a __Cocos3D__ based 3D renderer to officially introduce 3D support. The addition of 3D features can greatly enrich the expressiveness of 2D games, reduce the resource overhead of 2D games, and bring more imagination and ideas to expand games.
+However, it should be emphasized that the target users of __Cocos Creator__ is still 2D game development, and the **2D game is still the core** service of __Cocos Creator 2.x__. We will continue to optimize the development experience and basic performance of 2D games, and gradually increase the 3D functions suitable for integration into 2D games. Therefore, __Cocos Creator__ does not allow a pure 3D game, which is the goal of __Cocos3D__. __Cocos Creator's__ development path is closer to 2.5D.
```

Comment:
1. ...is still 2D game development -> ...is still 2D game development team
2. the **2D game is still the core** service of **Cocos Creator 2.x** -> **the 2D game is still the core service goal of Cocos Creator 2.x era**.
3. Cocos Creator does not allow a pure 3D game -> Cocos Creator will not become a pure 3D engine
4. development path -> roadmap

Reply:
1. I have to add _the_ now.

2. we dont use *era* like this. *era* to us is a long period of time. I used *releases*

3. Yes, that sounds better. Thank you.

4. fixed.